packages/azure-theme/package.json

```diff
@@ -15,10 +14,12 @@
   },
   "peerDependencies": {
     "@fluentui/react-components": ">=9.46.3 <10.0.0",
-    "@fluentui/react-icons": ">=2.0.204 <3.0.0",
-    "@types/react": ">=16.8.0 <19.0.0",
-    "@types/react-dom": ">=16.8.0 <19.0.0",
     "react": ">=16.8.0 <19.0.0",
     "react-dom": ">=16.8.0 <19.0.0"
+  },
+  "devDependencies": {
+    "@fluentui/react-icons": ">=2.0.204 <3.0.0",
+    "@types/react": ">=16.8.0 <19.0.0",
+    "@types/react-dom": ">=16.8.0 <19.0.0"
```

Comment:
@adr1-jpg 

You should not move react dependencies away from peer deps. Regarding icons, I think it should be either peer dep or normal dependency as it was